### PR TITLE
WIP: test azure-pipelines

### PR DESCRIPTION
**Pull request by iMichka**

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -1,19 +1,63 @@
...
+    git config --global user.name LinuxbrewTestBot
+    git config --global user.email testbot@linuxbrew.sh
+    PATH="/home/linuxbrew/.linuxbrew/bin:/home/linuxbrew/.linuxbrew/sbin:$PATH"
+    sudo rm -rf /home/linuxbrew/.linuxbrew/Homebrew/Library/Taps/homebrew/homebrew-core
+    sudo cp -a $(Build.Repository.LocalPath) /home/linuxbrew/.linuxbrew/Homebrew/Library/Taps/homebrew/homebrew-core
+    cd /home/linuxbrew/.linuxbrew/Homebrew/Library/Taps/homebrew/
+    sudo git clone https://github.com/iMichka/homebrew-test-bot homebrew-test-bot
+    cd homebrew-test-bot
+    sudo git fetch origin
+    sudo git checkout azure
```
Comment (maxim-belkin) [inline azure-pipelines.yml]: why sudo?

Comment (iMichka) [reply azure-pipelines.yml]: Maybe I should put the `sudo chown $(whoami): -R /home/linuxbrew` earlier.
The `linuxbrew` users is now not the default one anymore, it is `root`.

Comment (maxim-belkin) [reply azure-pipelines.yml]: if default user is root, then you don't need `sudo` at all, no?

Comment (iMichka) [reply azure-pipelines.yml]: I need to have a closer look. Got a permission denied, this is why I added the sudo calls.

**Pull request closed without merging**